File: tests/fixtures/sample.lua
local function add(a, b)
    return a + b
end

function greet(name)
    return "Hello, " .. name .. "!"
end

local M = {}

function M.multiply(a, b)
    return a * b
end

function M.divide(a, b)
    return a / b
end

return M

File: tests/fixtures/sample.ex
defmodule Calculator do
  def add(a, b) do
    a + b
  end

  defp private_add(a, b) do
    a + b
  end

  def multiply(a, b) do
    a * b
  end
end

defmodule Greeter do
  def greet(name) do
    "Hello, #{name}!"
  end
end
